code/game/objects/items/melee/f13onehanded.dm

```diff
@@ -155,6 +155,7 @@
 	bare_wound_bonus = 5
 	custom_materials = list(/datum/material/iron=6000)
 	resistance_flags = FIRE_PROOF
+	attack_speed = CLICK_CD_MELEE * 0.85 //6.8
```

Comment:
Id prefer the 0.85 and such to also be defines, so they're easier to change later if needed. You can put them in combat.dm's defines!

Reply:
Same for the armor penetration values.

Reply:
The reason I didn't make them defines was because a lot of them have smaller tweaks. It feels like it'd be redundant and just weird. I will do it if you _really_ want to, but I don't think it's needed.

Reply:
Should be fine for now then!